### PR TITLE
added integration of video and play dashboard

### PR DESCRIPTION
**Pull request by usmangt**
added integration of video and play dashboard so that the video link and also the play demo dashboard is visible inside the Plugin catalog once the user views this plugin.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,6 +10,14 @@ Visualize your Google Spreadsheets with Grafana
 
 For the plugin documentation, visit [plugin documentation website](https://grafana.com/docs/plugins/grafana-googlesheets-datasource/).
 
+## Video Tutorial
+
+Watch this video to learn more about setting up the Grafana Google Sheets data source plugin:
+
+[![Google Sheets data source plugin | Visualize Spreadsheets using Grafana | Tutorial](https://img.youtube.com/vi/hqeqeQFrtSA/hq720.jpg)](https://youtu.be/hqeqeQFrtSA "Grafana Google Sheets data source plugin")
+
+{{< docs/play title="Google Sheets data source plugin demo" url="https://play.grafana.org/d/ddkar8yanj56oa/visualizing-google-sheets-data" >}}
```
Comment (zoltanbedi) [inline README.md:19]: Why is this needed? This looks weird in GitHub when you look at the readme.

**Review by usmangt — COMMENTED** [review]
done the changes

**Review by zoltanbedi — APPROVED** [review]
lgtm
